protocols/v2/roles-logic-sv2/src/handlers/job_negotiation.rs

```diff
@@ -6,22 +6,24 @@ use crate::errors::Error;
 use core::convert::TryInto;
 use job_negotiation_sv2::SetCoinbase;
 
+/// A trait implemented by a downstream to handle SV2 job negotiation messages.
 pub trait ParseServerJobNegotiationMessages
 where
     Self: Sized,
 {
+    // Used to parse job negotiation message and route to the message's respected handler function
```

Comment:
Was this intentionally `//`

Reply:
This was unintentional. The `pub trait` makes those functions public